lib/locale.ts

```diff
@@ -55,6 +55,7 @@ export type ITranslationKeys =
   | "character-dialog.template.IronEddaAccelerated"
   | "character-dialog.template.Maze"
   | "character-dialog.template.TheWitchIsDead"
+  | "character-dialog.template.EdgeOfTheEmpire"
```

Comment:
**question** @Gpoitras I'm curious, did you add this line manually @Gpoitras or did you use the i18n npm task ?